src/Streamly/Internal/System/Process.hs

```diff
@@ -442,11 +438,11 @@ pipeBytes' ::
     => FilePath         -- ^ Executable name or path
     -> [String]         -- ^ Arguments
     -> Stream m Word8        -- ^ Input Stream
-    -> Stream m (Either Word8 Word8) -- ^ Output Stream
+    -> Stream m (Word8) -- ^ Output Stream
```

Comment:
We cannot change the signature of this function. It has changed the entire behavior of this API, it's purpose is gone. See the documentation above. It returns the Either type to distinguish error from regular output.